code/modules/mob/living/simple_animal/abnormality/aleph/titania.dm

```diff
@@ -295,14 +312,145 @@
+			F.AddToPlayer(src)
+			to_chat(victim, span_userdanger("A fairy is orbiting you!"))
+	. = ..()
+
+/datum/status_effect/fairy_lights
+	id = "fairy_lights"
+	alert_type = /atom/movable/screen/alert/status_effect/fairy_lights
+	duration = 30 SECONDS // Hits 15 times
+	tick_interval = 2 SECONDS
+	var/list/fairies = list()
+	var/limit = 4
+	var/current_overlay = ""
+
+/atom/movable/screen/alert/status_effect/fairy_lights
+	name = "Fairy Lights"
+	desc = "Fairies orbiting around you is causing you to take WHITE damage!"
+	icon = 'ModularTegustation/Teguicons/status_sprites.dmi'
+	icon_state = "fairy_lights"
+
+/datum/status_effect/fairy_lights/on_apply()
+	. = ..()
+	if(!owner)
+		return
+	RegisterSignal(owner, COMSIG_MOB_APPLY_DAMGE, PROC_REF(HurtFaries))
+
+/datum/status_effect/fairy_lights/on_remove()
+	. = ..()
+	if(!owner)
+		return
+	if(fairies.len > 0)
+		for(var/mob/living/simple_animal/hostile/fairyswarm/fairy in fairies)
+			fairy.toggle_ai(AI_ON)
+			fairy.forceMove(get_turf(owner))
+			fairy.status = null
+			fairies -= fairy
+	if(ishuman(owner))
+		owner.cut_overlay(mutable_appearance('icons/effects/32x64.dmi', current_overlay, -FIRE_LAYER))
+	UnregisterSignal(owner, COMSIG_MOB_APPLY_DAMGE)
+
+/datum/status_effect/fairy_lights/proc/AddToPlayer(mob/living/simple_animal/hostile/fairyswarm/F)
+	if(F)
+		fairies += F
+		F.forceMove(owner)
+		F.status = src
+		F.toggle_ai(AI_OFF)
+		UpdateOverlay()
+
+/datum/status_effect/fairy_lights/proc/UpdateOverlay()
+	if(fairies.len < 1)//to prevent overlays from sticking
+		return
+	if(fairies.len > 1)
+		linked_alert.desc = "[fairies.len] fairies are orbiting around your head, causing you to take WHITE damage!."
+	else
+		linked_alert.desc = "A fairy is orbiting around your head, causing you to take WHITE damage!."
+	if(ishuman(owner))
+		var/old_overlay = current_overlay
+		current_overlay = "fairy_lights_[fairies.len]"
+		var/mob/living/simple_animal/hostile/fairyswarm/F = fairies[1]
+		if(F)
+			if(F.icon_state == "fairyswarm_oberon")
+				current_overlay += "_oberon"
+		owner.cut_overlay(mutable_appearance('icons/effects/32x64.dmi', old_overlay, -FIRE_LAYER))
+		owner.cut_overlay(mutable_appearance('icons/effects/32x64.dmi', current_overlay, -FIRE_LAYER))
+		owner.add_overlay(mutable_appearance('icons/effects/32x64.dmi', current_overlay, -FIRE_LAYER))
+
+/datum/status_effect/fairy_lights/proc/HurtFaries(datum/source, damage, damagetype, def_zone)
+	if(!owner || damagetype != RED_DAMAGE)//I know in wonderlab it says red or white but if it was white than fairies could kill the fairies inside you
+		return
+	if(!damage || damage <= 20)
+		return
+	if(fairies.len > 1)
+		to_chat(owner, span_nicegreen("The hit killed all of the fairies orbiting you."))
+	else
+		to_chat(owner, span_nicegreen("The hit killed the fairy orbiting you."))
+		for(var/mob/living/simple_animal/hostile/fairyswarm/F in fairies)
+			F.Destroy()
```

Comment:
Do not do this, ever. `Destroy()` and `Initialize()` are procs that should never be directly called unless you want to badly break stuff
```suggestion
			qdel(F)
```